interview-questions.md

```diff
@@ -5,57 +5,115 @@ Answer the following questions. First, without external resources. Challenge you
 1. Why would you use the method super()?
 
   Your answer:
+  You would use the method super() in react if a state is needed for a component. You also use it inside a constructor for a class.
```

Comment:
Great! Its a bit confusing though, as the 'interviewer' is not specifically asking about React and Super is used with JS classes as well.